app/views/components/_related-navigation.html.erb

```diff
@@ -58,18 +59,18 @@
                   %>
                 <% end %>
               <% end %>
-              <% if links.length > max_section_length %>
-              <li class="app-c-related-navigation__link toggle-wrap">
-                <a href="#"
-                  data-controls="toggle_<%= section_title %>"
-                  data-expanded="false"
-                  data-toggled-text="<%= t("govuk_component.metadata.toggle_less", default: "Show fewer") %>">
-                    <%= t("govuk_component.metadata.toggle_more", number: remaining_links(links, max_section_length), default: "+ #{remaining_links(links, max_section_length)} more") %>
-                </a>
-              </li>
-              <li class="app-c-related-navigation__link">
-                <span id="toggle_<%= section_title %>" class="js-hidden"><%= constructed_link_array.to_sentence.html_safe %></span>
-              </li>
+              <% if links.length > section_link_limit %>
```

Comment:
Personally I would have just gone with 
<% if links.length > section_link_limit + 1 %>
possibly with a comment to explain why, rather than adding quite verbose extra logic, would be good to hear other opinions though.

Reply:
Do you mean instead of having the `calculate_section_link_limit` method?

Reply:
Yes, it just felt like a lot of overhead for effectively a + 1 in the loop initialisation.

Reply:
@maxgds I think this is what I initially tried, but it didn't work and the logic was actually more complicated than I thought. Totally happy to change if there is a simpler way though, perhaps I'm missing something.

Reply:
Oh, don't worry then if it doesn't work, it was just an assumption on my part.